packages/phoenix-evals/src/phoenix/evals/models/bedrock.py

```diff
@@ -134,7 +128,7 @@ def _rate_limited_completion(self, **kwargs: Any) -> Any:
         @self._rate_limiter.limit
         def _completion(**kwargs: Any) -> Any:
             try:
-                return self.client.invoke_model(**kwargs)
+                return self.client.converse(**kwargs)
```

Comment:
I'd pop `instruction` from the kwargs just to make sure it doesn't leak into the new API (this is the system instruction, let's not support it for now since not all models support this)